app/helpers/application_helper/toolbar/miq_policy_center.rb

```diff
@@ -26,7 +26,7 @@ class ApplicationHelper::Toolbar::MiqPolicyCenter < ApplicationHelper::Toolbar::
           end,
           :confirm   => proc do
                           _("Are you sure you want to create Policy [%{new_policy_description}] from this Policy?") % {
-                            :new_policy_description => truncate("Copy of \#{@policy.description}\", :length => 255, :omission => \"")
+                            :new_policy_description => truncate("Copy of \#{@policy.description}", :length => 255, :omission => "")
```

Comment:
I think the \# is still wrong.
And the bigger problem is `truncate` doesn't resolve.